src/jit/target.h

```diff
@@ -1884,11 +1892,58 @@ inline bool         genIsValidDoubleReg(regNumber reg)
 #endif // defined(LEGACY_BACKEND) && defined(_TARGET_ARM_)
 
 /*****************************************************************************
+*
+*  Returns true if our target architecture uses a fixed return buffer register
+*/
+inline bool         hasFixedRetBuffReg()
+{
+    // Disable this until the VM changes are also enabled
+#if 0 //def _TARGET_ARM64_
+    return true;
+#else
+    return false;
+#endif
+}
+
+/*****************************************************************************
+*
+*  Returns the regNumber for the fixed return buffer 
+*/
+inline regNumber    theFixedRetBuffReg()
+{
+    assert(hasFixedRetBuffReg());   // This predicate should be checked before calling this method
+#ifdef _TARGET_ARM64_
+    return REG_ARG_RET_BUFF;
+#else
+    return REG_NA;
+#endif
+}
+
+/*****************************************************************************
```

Comment:
As per Jit CC: function header would use 

```
//-------------------------------------------------------------------------------------------
// theFixedRetBuffArgNum: Returns the argNum for the fixed return buffer
//
```

Instead of C-style 
```
/**************
 *
*/
```